### PR TITLE
Covid deliveries

### LINKED ISSUE #131: Covid-19: Deliveries and takeout
Add the information about deliveries and takeout to the popup.

Example data: https://features.caresteouvert.fr/collections/public.poi_osm/items/n7017438374.json

This is how bleibt-offen visualises it.

Screenshot from 2020-04-16 11-17-05

### PR DESCRIPTION
**Pull request by meromisi**
- added delivery and take-away information to popup window
- fix OSM opening hours
- added CSS

closes #131 
image

### CODE REVIEW

File: app/component/map/popups/OSMOpeningHours.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import SimpleOpeningHours from 'simple-opening-hours';
+import { SimpleOpeningHours } from 'simple-opening-hours';
```
Comment (leonardehrenfried) [inline app/component/map/popups/OSMOpeningHours.js:1]: This needs to be reverted. Run `yarn install`.

File: app/component/map/popups/OSMOpeningHours.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default class OSMOpeningHours extends React.Component {
   render() {
     const opening = new SimpleOpeningHours(this.props.openingHours);
     const { displayStatus } = this.props;
-    const isOpenNow = opening.isOpen();
+    const isOpenNow = opening.isOpenNow();
```
Comment (leonardehrenfried) [inline app/component/map/popups/OSMOpeningHours.js:61]: This one too.

**Comment by leonardehrenfried** [comment]
Please add the following translations:

- Take-away service available/Mitnehmen möglich
- Delivery service available/Lieferservice möglich

**Comment by meromisi** [comment]
@leonardehrenfried is it possible that the objects don't have `takeaway`, `delivery`, `deliver:covid19` or `takeaway:covid19` properties? We've checked a lot of restaurants and only found a `takeaway:ice_cream` property.

**Comment by leonardehrenfried** [comment]
Yes, there is very little data in Herrenberg. The local OSM community or the city itself have to add it.

**Comment by leonardehrenfried** [comment]
Can you please take care of my review comments?

**Comment by Beck-berry** [comment]
@leonardehrenfried That commit is reverted by 473acaa, isn't it?

**Comment by leonardehrenfried** [comment]
You're right. :man_facepalming:

**Pull request merged**